sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -137,20 +130,18 @@
     "frequency/@units": "@attrs:metadata/scientificMetadata/Source/repetition_rate/unit",
     "associated_beam": "/entry/instrument/beam_probe"
   },
-  "/ENTRY[entry]/INSTRUMENT[instrument]/beam_TYPE[beam_probe]": {
+  "/ENTRY[entry]/INSTRUMENT[instrument]/beamTYPE[beam_probe]": {
     "distance": 0.0,
     "distance/@units": "mm",
     "incident_energy": "@attrs:metadata/scientificMetadata/Source/photon_energy/value",
-    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Source/dispersion/unit",
-    "incident_energy_dispersion": "@attrs:metadata/scientificMetadata/Source/dispersion/value",
-    "incident_energy_dispersion/@units": "@attrs:metadata/scientificMetadata/Source/dispersion/unit",
-    "pulse_duration": "t.b.d.",
+    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Source/photon_energy/unit",
+    "pulse_duration": 100,
     "pulse_duration/@units": "fs",
-    "incident_polarization": "horizontal",
-    "incident_polarization/@units": "None",
+    "incident_polarization": [1, 1, 0, 0],
+    "incident_polarization/@units": "V^2/mm^2",
```

Comment:
Polarization as stokes vectors. Should be converted from SciCat values if possible.

Reply:
We have just the angle of the quarter wave plate, where e.g. 0 degrees = s-polarization, 45 degrees = p-polarization. How can I get to those vectors? :)

Reply:
This is also something you can do in metadata generation from the WP angle.
This is maybe helpful.
For horizontal, it is (1, 1, 0, 0,), for vertical (1, -1, 0, 0), for any linear angle (1, cos2Theta, sin2Theta, 0), where Theta is the linear rotation angle of the light (twice the WP angle). 
For combined l/2 l/4 plates it becomes more complcated.